cmd/test/cmd.go

```diff
@@ -288,18 +264,18 @@ func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtr
 			Namespace:        options.Namespace,
 			K8sContext:       options.K8sContext,
 			Variables:        options.Variables,
-			Build:            false,
+			Build:            true,
 			Dependencies:     false,
 			Timeout:          options.Timeout,
 			RunWithoutBash:   false,
-			RunInRemote:      manifest.Deploy.Remote,
+			RunInRemote:      runInRemote,
 			Wait:             true,
 			ShowCTA:          false,
 		})
-		c.TrackDeploy(manifest, manifest.Deploy.Remote, deployStartTime, err)
+		c.TrackDeploy(manifest, runInRemote, deployStartTime, err)
 
 		if err != nil {
-			oktetoLog.Error("deploy failed: %s", err.Error())
+			oktetoLog.Errorf("deploy failed: %s", err.Error())
```

Comment:
this was displaying broken:

<img width="872" alt="Screenshot 2024-05-27 at 19 34 03" src="https://github.com/okteto/okteto/assets/2318450/a3dd2b53-5d32-4c1b-b39c-14332ccfbcca">

it's also redundant, so we may consider remove it. Thoughts?